### DISCUSSION TITLE
Issues with keyboard on GC on 1.8.4 HW2

### DISCUSSION THREAD

**Discussion by rynoweiss** [darthcloud/BlueRetro#775] [Q&A]
I have the LaserBear internal mod. On v1.8.4_hw2/BlueRetro_gamecube.bin, I experience very weird behavior with Keyboard. It mashes the letter f. If you hold down any key, it stops mashing f, and any other keys you type will be typed accurate. After doing a factory reset, which reverts to v1.8.1, the keyboard behavior becomes normal. Keyboard I am using is a Logitech K380, and the game I am trying to play with keyboard is PSO.

**Comment by darthcloud** [comment]
This got broken on the first commit right after v1.8.3 way back in February when I merged PR #620.

**Comment by darthcloud** [comment]
[v1.8.4-17-g20c2677_hw1.zip](https://github.com/darthcloud/BlueRetro/files/12764774/v1.8.4-17-g20c2677_hw1.zip)
[v1.8.4-17-g20c2677_hw2.zip](https://github.com/darthcloud/BlueRetro/files/12764776/v1.8.4-17-g20c2677_hw2.zip)

Can you try those build, it fix it for me.
